dev/src/protos/protos.json

```diff
@@ -3973,8 +3973,7 @@
                 "REQUIRED": 2,
                 "OUTPUT_ONLY": 3,
                 "INPUT_ONLY": 4,
-                "IMMUTABLE": 5,
-                "UNORDERED_LIST": 6
```

Comment:
Can you verify that this change was meant to happen? I am surprised that there is no open PR from Synthtool if this is an intended change.

The source of truth might be here: https://source.corp.google.com/piper///depot/google3/google/api/field_behavior.proto;l=56;bpv=1

Reply:
I'm not sure what I've done differently. When I run synthtool on a master, this file isn't changed. However, I've verified that running synthtool on this branch results in this change happening. Tests still pass, but I'm not sure if this is a sign of something else wrong.